### DISCUSSION TITLE
The CHIA program has shut down and can not see the plots now.

### DISCUSSION THREAD

**Discussion by koza961** [Chia-Network/chia-blockchain#4110] [General]
Hello.
CHIA has shutdown itself.
Cannot see my plots when turned on.

There is only information:
Failed to cultivate (incorrect beds)
These parcels are invalid, you may want to delete them.

Can you fix it somehow?

**Comment by SackofFluor** [comment]
easiest way to check whether its still working is to use the taskmanager or resource manager.

if there is still a high usage its all running fine in the background.

If the plotting was almost done you may even be able to salvage the plot.

**Reply by garyhkirk** [reply]
Are the plots in question just the ones in progress or have you lost sight of completed plots?

**Reply by koza961** [reply]
All plots have been lost.
I turned off the computer.
I turned it on after an hour and it can already see the plots :)
